billy/web/api/handlers.py

```diff
@@ -399,20 +400,31 @@ def read(self, request):
         boundary_mapping = {}
         jurisdiction = None
 
-        for dist in resp['objects']:
-            # look up district slug
-            boundary_id = dist['url'].replace('/boundaries/', '').strip('/')
-            districts = db.districts.find({'boundary_id': boundary_id})
+        for dist in resp['results']:
+            ocdid = dist['id']
+            # ocd-division/country:us/state:oh/cd:11
+            _, localpart = ocdid.rsplit("/", 1)
+            set_, series = localpart.split(":", 1)
+            if set_ not in ['sldl', 'sldu']:
+                # Place, CD, County, ...
+                continue
+
+            districts = db.districts.find({'division_id': ocdid})
```

Comment:
`division_id` is basically the new `boundary_id`. Leaving `boundary_id` as is until after the migration is complete.